src/features/content-explorer/messages.js

```diff
@@ -1,11 +1,6 @@
 import { defineMessages } from 'react-intl';
 
 const messages = defineMessages({
-    bookmark: {
```

Comment:
are we moving these because we plan to use them for ContentPicker?

Reply:
I moved them out since they're not used in `features/` anymore (after the `ItemListIcon.js` refactor). I also didn't see ContentPicker in `features/`

Reply:
ah okay i see what's happening now. I thought this was happening in the elements version. I don't think we should be importing things from `/elements` into `/features`. we probably should keep these separate and almost think of them as separate packages

Reply:
@greg-in-a-box thoughts on importing things from the `/elements` folder? my gut is telling me we should avoid that so things do not become dependent on `/elements`

Reply:
Yeah, I'm with you. I decided to go with it anyway since I saw it done in a couple other places:
<img width="470" alt="image" src="https://user-images.githubusercontent.com/34158284/216458437-62b564a5-6421-4af8-9394-7525dcc48aa0.png">

And it's a tradeoff with duplicated code. If we want to avoid importing, then I'll go ahead and undo some of the refactor in `ItemListIcon.js`

Reply:
I think you can keep the changes. I thought it was a lot more (like all the messages as well) but it seems it's only for `getIcon` inside of `ItemListIcon`